tpv24/tpv24_fault.yaml

```diff
@@ -8,15 +8,15 @@
   function: |
       function f (x)
         b22 = 0.926793
-        b23 = -0.169029
         b33 = 1.073206
+        b23 = -0.169029
         z = x["z"]
-        s_zz = 2670*9.8*z
-        Pf = -1000*9.8*z
+        s_zz = 2670.0 * 9.8 * z
+        p_f = -1000.0 * 9.8 * z
         if (z >= -15600.0) then
-          s_xx = b22 * (s_zz + Pf) - Pf
-          s_xy = b23 * (s_zz + Pf) - Pf
```

Comment:
Only this line (s_xy) has to be corrected:
![image](https://github.com/SeisSol/Examples/assets/13536910/e60a52f0-5d4c-4e61-b76d-4a34cf4a8819)
https://strike.scec.org/cvws/download/TPV24_25_Description_v07.pdf

Reply:
You're right. Thanks!